src/gui/hist_dialog.c

```diff
@@ -217,12 +217,14 @@ int dt_gui_hist_dialog_new(dt_gui_hist_dialog_t *d, int imgid, gboolean iscopy)
     do
     {
       dt_history_item_t *item = (dt_history_item_t *)items->data;
-
-      gtk_list_store_append(GTK_LIST_STORE(liststore), &iter);
-      gtk_list_store_set(GTK_LIST_STORE(liststore), &iter, DT_HIST_ITEMS_COL_ENABLED,
+      // This is sort of a hack, any better solution?
```

Comment:
Maybe we can test if commit_params() is present or NULL. The module gamma has no parameter and so no commit_params() routine, in such a case we know that it cannot be modified. At least we know that there is nothing to copy. What do you think?

Reply:
maybe just remove all IOP_FLAGS_HIDDEN iops?

Reply:
My proposal is not simple as at this point we only have an history item with no module information at all.

Reply:
@parafin : that would remove gamma, finalscale, mask_manager, overexposed & rawoverexposed. Sounds like it will work yes. But again, at this point we don't have the module information, just the module name. So we need first to get the module given the name and check the flags. Would be the best solution.

Reply:
Let me add a new routine get_module_by_name() to make this possible.

Reply:
Yes. I chose the flag testing version